### DISCUSSION TITLE
getDecodedData will reduce the sound quality

### DISCUSSION THREAD

**Discussion by XiiiRuth** [katspaugh/wavesurfer.js#3501] [Q&A]
## Bug description
When I wanted to export region as a file from MP3 music, I used this method(getDecodedData) to get ArrayBuffer.But the sound quality of the exported file has decreased.After trying many, many ways,I used AudioContext(). decodeAudioData to get the ArrayBuffer of the source audio.And without any modifications, the sound quality improved

## Environment
 - Browser: Chrome

**Comment by katspaugh** [comment]
wavesurfer's getDecodedData isn't meant for exporting audio, it's only for saving peaks for rendering.
You can, however, initialize wavesurfer with a higher sampleRate in the options (the default is 8000).
